src/librustdoc/clean/types.rs

```diff
@@ -1082,9 +1082,6 @@ impl Attributes {
         let mut out = String::new();
         add_doc_fragment(&mut out, ori);
         for new_frag in iter {
-            if new_frag.kind != ori.kind || new_frag.parent_module != ori.parent_module {
-                break;
-            }
```

Comment:
Why is this change in here? I think you must have pushed up an extra commit by mistake.

Reply:
Yes, this is a mistake. I will fix it.